compaction/task_manager_module.cc

```diff
@@ -549,7 +522,29 @@ future<> shard_reshaping_compaction_task_impl::run() {
 future<> table_resharding_compaction_task_impl::run() {
     auto all_jobs = co_await collect_all_shared_sstables(_dir, _db, _status.keyspace, _status.table, _owned_ranges_ptr);
     auto destinations = co_await distribute_reshard_jobs(std::move(all_jobs));
-    co_await run_resharding_jobs(_dir, std::move(destinations), _db, _status.keyspace, _status.table, std::move(_creator), std::move(_owned_ranges_ptr));
+
+    uint64_t total_size = boost::accumulate(destinations | boost::adaptors::transformed(std::mem_fn(&replica::reshard_shard_descriptor::size)), uint64_t(0));
+    if (total_size == 0) {
+        co_return;
+    }
+
+    auto start = std::chrono::steady_clock::now();
+    dblog.info("Resharding {} for {}.{}", utils::pretty_printed_data_size(total_size), _status.keyspace, _status.table);
```

Comment:
should it still use dblog?
not the compaction logger?

Reply:
We discussed that offline and decided to keep it this way for the time being. This is consistent with tasks previously moved here already.